modules/ROOT/pages/rtc-supported-functionality.adoc

```diff
@@ -18,6 +18,10 @@ The Real-Time Collaboration (RTC) plugin supports the latest desktop versions of
 
 Real-Time Collaboration is currently not supported on mobile devices.
 
+== IME support
+
+The Real-Time Collaboration (RTC) plugin currently doesn't support Input Method Editor (IME) as a input method.
```

Comment:
I think keep it in the same phrasing as the above mobile devices, rather than the below tinymce features.

```suggestion
== Input support

Real-Time Collaboration does not currently support using an Input Method Editor (IME).
```

Reply:
I just felt that it was a bit repetitive. But I changed it to the suggested text.